docs/for-developers/technical-overview.md

```diff
@@ -3,71 +3,200 @@ id: technical-overview
 title: Technical Overview
 ---
 
-## The Software Architecture of Drips
+This page offers an introduction to the core mechanics of the Drips smart contracts. The implementation can be found on Github:
 
-The diagram below provides an overview of how Drips works under the hood. Drips is a 100% decentralized protocol that is built 100% on open source software and tools.
-  
-![Architecture][ac]
-  
-In the following sections, we'll dive deeper into each of the different components of Drips and explain the role that each one plays.
+https://github.com/radicle-dev/radicle-drips-hub
 
-Before diving in however, an important point we would like to make is that Drips is entirely modular and there is no requirement for users to interact with the components in the way they are arranged above. For example, the Drips Webapp is offered only as a reference implementation and users are free to create their own web apps or even interact with the smart contracts directly using transactions issued directly using Etherscan or similar tools. Similarly, there is nothing to stop users from querying the Radicle/Drips subgraph and IPFS files using any applications or tools they desire. By designing Drips in this way, the goal of our team has been to build and offer up the world's most flexible and powerful open protocol for recurring payments and fundraising of any kind. With that in mind, our hope is that many future apps, protocols and tools will be able to build on Drips and use it to empower solutions we can't even imagine today.
+## Drips Concepts
 
-## Accounts and Balances
+### Drips - The Protocol
 
-For a deep dive into the details of how streaming is managed behind the scenes in DripsHub, please see <a href="https://github.com/radicle-dev/radicle-drips-hub/blob/master/docs/how_the_pool_works.md">here</a>.
+Drips is an Ethereum protocol that allows users to set up and manage streaming payments - continuous transfers of funds from one account to
+another over time. We refer to such streaming payments as "Drips". In some ways, streaming payments are similar to traditional recurring payments, but in other ways they are quite different and offer unique advantages.
 
-## Ethereum Contracts
+![](https://i.imgur.com/L3XSop5.png)
 
-### The DaiDripsHub Smart Contract
+Technically, tokens that are Dripped are not sent directly to the recipient's address. Instead the `DripsHub` contract keeps track of the sender and recipient's balances and allows the receiver to collect funds whenever they wish.
```

Comment:
(The word "address" makes me think of the `AddressApp`)

Reply:
For sure. Will address that in a separate docs page